src/Razor/test/Microsoft.VisualStudio.LanguageServerClient.Razor.Test/HtmlCSharp/DocumentHighlightHandlerTest.cs

```diff
@@ -67,60 +67,74 @@ public async Task HandleRequestAsync_ProjectionNotFound_ReturnsNull()
         }
 
         [Fact]
-        public async Task HandleRequestAsync_HtmlProjection_ReturnsNull()
+        public async Task HandleRequestAsync_CSharpProjection_RemapsHighlightRange()
```

Comment:
Git diff isn't being rendered ideally here. Main diff in this file is the removal of `HandleRequestAsync_HtmlProjection_ReturnsNull` and addition of `HandleRequestAsync_HtmlProjection_RemapsHighlightRange`